_posts/2019-03-01-announcing-the-moveit-2-port.md

```diff
@@ -0,0 +1,34 @@
+---
+author: robroboto
+comments: false
+date: 2019-03-01
+layout: post
+title: Announcing the MoveIt2 Port
+media_type: image
+media_link: /assets/images/sponsors/acutronic.jpg
+description: Exciting news – the MoveIt! port to ROS 2.0 has begun! We are happy to announce that [Acutronic Robotics](https://acutronicrobotics.com/) is partnering with [PickNik](https://picknik.ai/) to sponsor a MoveIt! codesprint to begin the port to ROS 2.
+categories:
+- MoveIt!
+- ROS
+---
+
+Exciting news – the MoveIt! port to ROS 2.0 has begun! We are happy to announce that [Acutronic Robotics](https://acutronicrobotics.com/) is partnering with [PickNik](https://picknik.ai/) to sponsor a MoveIt! codesprint to begin the port to ROS 2.
+
+[<img src="{{ site.url }}/assets/images/sponsors/acutronic.jpg" width="500" style="margin-right:20px"/>]({{ site.url }}/assets/images/sponsors/acutronic.jpg)
```

Comment:
@nbbrooks actually this should be removed because its now duplicated:
https://moveit.ros.org/moveit!/ros/2019/03/01/announcing-the-moveit-2-port.html

We should probably make the PickNik website also auto-include the image on the page, so there are fewer differences between these two blogging approaches

Reply:
See https://github.com/ros-planning/moveit.ros.org/pull/264